### DISCUSSION TITLE
Add Icons for multiple images (with subcaptionbox or captionbox) and for Wrapfigure/WrapTables.

### DISCUSSION THREAD

**Discussion by rafa4mat** [WinEdt-Team/WinEdt-Support#88] [1.3 Ideas and features requests]
Two images in an icon.

A wrapped image in an Icon, similar to section Icon.

A wrapped table in an Icon, similar to section Icon.

**Comment by Alex-Simonic** [comment]
You can create extra png images at a size suitable for your resolution and place them into your local folder:

`    IMG_LOCAL_FOLDER="%b\Bitmaps\Images"
`

They will be loaded and available when you restart WinEdt...

**Reply by Alex-Simonic** [reply]
If by "wrapped icon" you mean a drop-down toolbar button, you must modify Toolbar.ini (after you create the associated menu items that insert what you want). The Section button (in a large 2-row Toolbar) is defined as:

```
  BUTTON="Section"
    TYPE=6
    MENU="Part"
    MENU="Chapter"
    MENU="-"
    MENU="Section"
    MENU="Subsection"
    MENU="Subsubsection"
    MENU="-"
    MENU="Paragraph"
    MENU="Subparagraph"
    MENU="-"
    MENU="Appendix"
```
I am not planning to add more such buttons to the defaults but you can add as many as you want.

**Comment by rafa4mat** [comment]
Thank you, Alex.

I tried, but no success!.

Putting the image at 

    c:\Program Files\WinEdt Team\WinEdt 11\Bitmaps\Images\Figures.png

restarting WinEdt11.2 and restarting Windows + WinEdt11.2  and can´t see the image Figures.png, even clicking "Reload Images" button 


![Figures](https://github.com/user-attachments/assets/8090ebc9-3278-403e-8bd8-daaac6de280c)

**Reply by Alex-Simonic** [reply]
You are not supposed to add anything to %B folder (you should know this by now with all your customizations). It should go to %b... Restarting Windows? C'mon...

**Comment by rafa4mat** [comment]
ok, Alex. Yes. It works fine.  Copying the file to 

    c:\Users\...\AppData\Roaming\WinEdt Team\WinEdt 11\Bitmaps\Images\Figures.png

![snap004377](https://github.com/user-attachments/assets/ae4682b8-3e99-4d76-998a-51319b34393c)

[Images-SubCaptions.zip](https://github.com/user-attachments/files/18244883/Images-SubCaptions.zip)
